.gitignore

```diff
@@ -15,3 +15,4 @@ config/settings/*.local.yml
 warnings*.log
 !/data/.keep
 log
+data/
```

Comment:
Does this conflict with what's on line 16?

Reply:
Yes. I'm removing line 16 so that files in /data are not retained.

Reply:
we need line 16.  That says, "nothing in data except the `.keep`" file.

Reply:
That would be my reading too, but it doesn't work that way. Without line 18, I get all of the contents of `data/`. However, since entirely unrelated to this ticket, I'm rolling it back and can work it out in a separate ticket.

Reply:
Great.